docs/velox-backend-support-progress.md

```diff
@@ -27,7 +27,7 @@ The total supported functions' number for [Spark3.3 is 387](https://spark.apache
 
 ### Operator Map
 
-Gluten supports 20 operators (Draw to right to see all data types)
+Gluten supports 28 operators (Draw to right to see all data types)
 
```

Comment:
Please add below operator support in the table
ShuffleExchangeExec: ColumnarShuffleExchangeExec in Gluten
ReuseExchange: ReuseExchange in Gluten
ExpandExec: ExpandExec in Gluten